pages/facts.html

```diff
@@ -1,12 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <script src="https://kit.fontawesome.com/1910a1aeff.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap" rel="stylesheet">
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Amazon Rainforest Facts</title>
+  <link rel="stylesheet" href="facts.css">
 </head>
 <body>
-  
+  <header class="wrapper">
+    <h1> AMAZON RAINFOREST FACTS </h1>
+      <div>
+        <nav>
```

Comment:
I don't see CSS rules targeting this `div`, what does the `div` around the `nav` give us? Can it be removed?